scripts/build/companion_libs/350-newlib_nano.sh

```diff
@@ -199,13 +199,13 @@ ENABLE_TARGET_OPTSPACE:target-optspace
 %rename cc1plus	newlib_nano_cc1plus
 
 *cpp:
--isystem ${CT_PREFIX_DIR}/newlib-nano/${CT_TARGET}/include %(newlib_nano_cpp)
+-isystem %:getenv(GCC_EXEC_PREFIX ../../newlib-nano/${CT_TARGET}/include) %(newlib_nano_cpp)
 
 *cc1plus:
--idirafter ${CT_PREFIX_DIR}/newlib-nano/${CT_TARGET}/include %(newlib_nano_cc1plus)  
+-idirafter %:getenv(GCC_EXEC_PREFIX ../../newlib-nano/${CT_TARGET}/include) %(newlib_nano_cc1plus)
```

Comment:
This had a minor conflict with a commit from @galak that removed the trailing space. I think I resolved it correctly.